buildconfig/CMake/Packaging/mantidpython.in

```diff
@@ -30,9 +30,14 @@ if [ -n "${PYTHONPATH}" ]; then
     LOCAL_PYTHONPATH=${LOCAL_PYTHONPATH}:${PYTHONPATH}
 fi
 
-# Launch
+if [ -n "$1" ] && [ "$1" = "--classic" ]; then
+    shift
+    PROG=$(command -v python)
+fi
+
+# Launch ipython or python
 LD_PRELOAD=${LOCAL_PRELOAD} TCMALLOC_RELEASE_RATE=${TCM_RELEASE} \
-    LD_LIBRARY_PATH=${LOCAL_LDPATH} QT_API=pyqt \
-    PV_PLUGIN_PATH=${PV_PLUGIN_PATH} \
-    PYTHONPATH=${LOCAL_PYTHONPATH} \
-    @WRAPPER_PREFIX@${IPYTHON} $*@WRAPPER_POSTFIX@
+LD_LIBRARY_PATH=${LOCAL_LDPATH} QT_API=pyqt \
```

Comment:
Sorry to be a pain but could we put the indents back in for this and the following 3 lines, like it was before? It just makes it a little easier to see at a glance that it's part of one command and not 4 separate ones.